CHANGELOG.md

```diff
@@ -1,33 +1,59 @@
-* On failing to open a file from Roblox Studio, it is now logged appropriately instead of being ignored ([#745]).
-* Significantly improved performance of `rojo sourcemap`. ([#668])
-* Fixed the diff visualizer of connected sessions. ([#674])
-* Fixed disconnected session activity. ([#675])
-* Skip confirming patches that contain only a datamodel name change. ([#688])
+### Additions
+* Added support for `.toml` files to `$path` ([#633])
+* Significantly improved performance of `rojo sourcemap` ([#668])
+* Add `plugin` flag to the `build` command that outputs to the local plugins folder ([#735])
+* Added `Terrain` classname inference, similar to services ([#771])
+
+	`Terrain` may now be defined in the projects without using `$classname`:
+	```json
+	"Workspace": {
+		"Terrain": {
+			"$path": "path/to/terrain.rbxm"
+		}
+	}
+	```
+
+* Added support for `Terrain.MaterialColors` ([#770])
+
+	`Terrain.MaterialColors` is now represented in projects in a human readable format:
+	```json
+	"Workspace": {
+		"Terrain": {
+			"$path": "path/to/terrain.rbxm"
+			"$properties": {
+				"MaterialColors": {
+					"Grass": [10, 20, 30],
+					"Asphalt": [40, 50, 60],
+					"LeafyGrass": [255, 155, 55]
+				}
+			}
+		}
+	}
+	```
+
+* Added better support for `Font` properties ([#731])
+* Add Confirmation Behavior setting ([#774])
+* Select Instances from diff tree view ([#709])
 * Added sync reminder notification. ([#689])
 * Added protection against syncing a model to a place. ([#691])
-* Select Instances from diff tree view ([#709])
-* Fix Rojo breaking when users undo/redo in Studio ([#708])
-* Improved sync info text on Connected page. ([#692])
-* Fix patch visualizer breaking when instances are removed during sync ([#713])
+* Add new plugin template to the `init` command ([#738])
+* Added rich Source diffs in patch visualizer ([#748])
 * Patch visualizer now indicates what changes failed to apply. ([#717])
 * Add buttons for navigation on the Connected page ([#722])
+* Improved sync info text on Connected page. ([#692])
+
+### Fixes
+* Fixed the diff visualizer of connected sessions. ([#674])
+* Fixed disconnected session activity. ([#675])
+* Skip confirming patches that contain only a datamodel name change. ([#688])
+* Fix Rojo breaking when users undo/redo in Studio ([#708])
 * Improve tooltip behavior ([#723])
 * Better settings controls ([#725])
-* Rework patch visualizer with many fixes and improvements ([#726])
-* Added support for syncing in `.toml` files ([#633])
-* Add `plugin` flag to the `build` command that outputs to the local plugins folder ([#735])
-* Added better support for `Font` properties ([#731])
-* Add new plugin template to the `init` command ([#738])
-* Added rich Source diffs in patch visualizer ([#748])
-* Fix PatchTree performance issues ([#755])
+* Rework patch visualizer with many fixes and improvements ([#713], [#726], [#755])
 * Don't override the initial enabled state for source diffing ([#760])
```

Comment:
This is an internal detail regarding Roact's weird interaction with plugin guis and doesn't need to be in the log at all imo